File: src/main/java/com/lanka_guide/tripplanner/model/activity/stay/HotelStay.java
package com.lanka_guide.tripplanner.model.activity.stay;

import com.lanka_guide.tripplanner.model.place.accommodation.Hotel;

import java.time.ZonedDateTime;

public class HotelStay extends Stay {
    Hotel hotel;

    public HotelStay(Hotel hotel, ZonedDateTime from, ZonedDateTime to) {
        super(from, to);
        this.hotel = hotel;
    }

}

File: src/main/kotlin/com/lankaguide/tripplanner/model/activity/stay/HotelStay.kt
package com.lankaguide.tripplanner.model.activity.stay

import com.lankaguide.tripplanner.model.place.accommodation.Hotel

import java.time.ZonedDateTime

class HotelStay(internal var hotel: Hotel, from: ZonedDateTime, to: ZonedDateTime) : Stay(from, to)
